src/H5ES.c

```diff
@@ -102,11 +103,62 @@ H5EScreate(void)
     FUNC_LEAVE_API(ret_value)
 } /* end H5EScreate() */
 
+/*-------------------------------------------------------------------------
+ * Function:    H5ESinsert_request
+ *
+ * Purpose:     Insert a request from a VOL connector into an event set.
+ *
+ * Note:        This function is primarily targeted at VOL connector
+ *              authors and is _not_ designed for general-purpose application use.
+ *
+ * Return:      SUCCEED / FAIL
+ *
+ * Programmer:	Quincey Koziol
+ *              Friday, December 11, 2020
+ *
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5ESinsert_request(hid_t es_id, hid_t connector_id, void *request)
```

Comment:
Should this include debugging parameters? Either explicitly or through a macro like the _async APIs?

Reply:
I trend in the "no" direction, since those debugging parameters are designed for application operations, and this API routine will only be invoked within a VOL connector.